packages/astro/src/integrations/astroFeaturesValidation.ts

```diff
@@ -103,17 +103,17 @@ function validateSupportKind(
 
 function featureIsUnsupported(adapterName: string, logger: Logger, featureName: string) {
 	logger.error(
-		`${adapterName}`,
-		`The feature ${featureName} is not supported by the adapter ${adapterName}.`
+		'config',
+		`The feature ${featureName} is not supported (used by ${adapterName}).`
```

Comment:
I'm not sure what the "used by" adapeter in parenthesis is supposed to convey here? I would be confused to see this message, for example:

> The feature confetti is not supported (used by Cloudflare).

Same goes for the ones below.